includes/utils.php

```diff
@@ -126,7 +126,7 @@ function set_meta( $post_id, $meta ) {
 			continue;
 		}
 
-		foreach ( $meta_values as $meta_placement => $meta_value ) {
+		foreach ( (array) $meta_values as $meta_placement => $meta_value ) {
```

Comment:
I think we are sending meta value as array anyway, why do we need to cast them here ? Maybe because of backward compatibility ?
https://github.com/NovemBit/distributor/blob/895feb9bf21a259bf4d7d26510c38dbf0b4bf24b/includes/utils.php#L294